engine/crates/telemetry/src/span/gql.rs

```diff
@@ -30,6 +35,8 @@ impl GqlRequestSpan {
             "gql.response.field_errors_count"  = Empty,
             "gql.response.data_is_null"  = Empty,
             "gql.response.request_errors_count"  = Empty,
+            "gql.response.latency_ms" = Empty,
```

Comment:
There is no point in adding `latency_ms` in a span, the span is _already_ an interval

Reply:
If you want it in logs, add it to the message OR the log message attributes when creating the `tracing` event. not the span.